Modules/TPC/run/tpcQCPadCalibration.json

```diff
@@ -115,8 +106,17 @@
         "dataSource" : [{
           "type" : "PostProcessing",
           "name" : "Calibration",
-          "MOs" : ["c_ROCs_Noise_1D","c_Sides_Noise","c_ROCs_Pedestal_2D"]
-        }]
+          "MOs" : ["c_ROCs_Noise_1D","c_Sides_Noise","c_ROCs_Pedestals_2D","c_ROCs_Noise_2D"]
+        }],
+        "stopTrigger_comment": [ "MOsNames are required for code to work, runs check of empty pads, should be the same as MOs, the MOs must be separated by comma only, no spaces" ],
+        "checkParameters": {
+          "mediumQualityPercentageOfWorkingPads" : "0.7",
+          "badQualityPercentageOfWorkingPads" : "0.4",
+          "mediumQualityNoiseMean" : "1.25",
+          "badQualityNoiseMean" : "1.5",
+          "MOsNames2D" : "c_ROCs_Pedestals_2D,c_ROCs_Noise_2D"
+        }
+        
```

Comment:
How about the PID and Track cluster check. Wasn't the idea to generalize the PadCalibrationCheck to be able to deal with all? If so, should the checks be added in this json?

Reply:
This check is supossed to work on all tasks which provide the canvas with 72 pads. It is already tested in PadCalibration and RawDigits (please, see corresponding json files in this PR). So I call the same check in all tasks which requires such generalized check.